kiali-server/values.yaml

```
@@ -39,6 +39,9 @@ deployment:
   image_pull_secrets: []
   image_version: ${HELM_IMAGE_TAG}
   ingress_enabled: true
+  logger:
+    log_level: info
+    log_format: text
```

trivial change - but all peer settings are sorted alphabetically here. So change the order. Also wrap string values in quotes, again for consistency.

```suggestion
    log_format: "text"
    log_level: "info"
```